.github/workflows/devshell.yml

```diff
@@ -18,27 +18,27 @@ jobs:
     steps:
       - name: Set ENV variables
         run: |
-          export PYTHONVERSION=`printf ${{ matrix.python }} | tail -c 1`
-          export PYTHONUSERBASE=${HOME}/python_packages
-          export PATH=${PYTHONUSERBASE}:${PATH}
-          export CC=${{ matrix.cc }}
-          export CONFIGURE_FLAGS="
+          gh_export() {
+            echo "$1<<__EOF__" >> $GITHUB_ENV
+            echo "$2"          >> $GITHUB_ENV
+            echo "__EOF__"     >> $GITHUB_ENV
+          }
```

Comment:
Can we add an `export $1=$2`, and use the named variables below? E.g.: `PYTHONVERSION`